CAP/lib/CAP/Model/Portals.pm

```diff
@@ -50,10 +50,15 @@ sub BUILD {
 
 sub portal_from_host {
   my ( $self, $host ) = @_;
```

Comment:
A reminder that $host could be uninitialized as well.  Checking if this is a string before doing string functions on it would be very helpful.